usr/sbin/pkcstok_migrate/pkcstok_migrate.c

```diff
@@ -0,0 +1,2298 @@
+/*
+ * COPYRIGHT (c) International Business Machines Corp. 2020
+ *
+ * This program is provided under the terms of the Common Public License,
+ * version 1.0 (CPL-1.0). Any use, reproduction or distribution for this
+ * software constitutes recipient's acceptance of CPL-1.0 terms which can be
+ * found in the file LICENSE file or at
+ * https://opensource.org/licenses/cpl1.0.php
+ */
+
+/*
+ * pkcstok_migrate - A tool for migrating token objects to 3.12 format.
+ *
+ */
+
+#define _GNU_SOURCE
+#include <dlfcn.h>
+#include <errno.h>
+#include <getopt.h>
+#include <memory.h>
+#include <linux/limits.h>
+#include <openssl/evp.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <termios.h>
+#include <unistd.h>
+#include <dirent.h>
+#include <pkcs11types.h>
+
+#include "sw_crypt.h"
+#include "defs.h"
+#include "host_defs.h"
+#include "local_types.h"
+#include "h_extern.h"
+#include "pkcs_utils.h"
```

Comment:
I would then add a #DEFINE OCK_TOOL here before the include of pkcs_utils.h

Reply:
Yep